apollo-federation/src/operation/mod.rs

```diff
@@ -259,6 +261,12 @@ impl PartialEq for SelectionSet {
 
 impl Eq for SelectionSet {}
 
+impl Hash for SelectionSet {
```

Comment:
Not to rehash this, but having this be a regular method and not a trait implementation would make me more comfortable.

Reply:
Underlying `SelectionMap` `Hash` is now order independent (matching `Eq`) and as a result contract between `Eq` and `Hash` of `SelectionSet` is uphold so why wouldn't we keep it as trait?

Reply:
Oh, that's right! That's what the ordering was for.